src/main/java/spireMapOverhaul/zones/frostlands/monsters/Spruce.java

```diff
@@ -0,0 +1,100 @@
+package spireMapOverhaul.zones.frostlands.monsters;
+
+import basemod.abstracts.CustomMonster;
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.animations.AnimateSlowAttackAction;
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.actions.common.DamageAction;
+import com.megacrit.cardcrawl.actions.common.GainBlockAction;
+import com.megacrit.cardcrawl.actions.common.RollMoveAction;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.localization.MonsterStrings;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.powers.StrengthPower;
+import spireMapOverhaul.SpireAnniversary6Mod;
+import spireMapOverhaul.util.Wiz;
+import spireMapOverhaul.zones.frostlands.powers.LeavePower;
+
+import static spireMapOverhaul.SpireAnniversary6Mod.modID;
+
+public class Spruce extends CustomMonster
+{
+    public static final String ID = SpireAnniversary6Mod.makeID("Spruce");
+    private static final MonsterStrings monsterStrings = CardCrawlGame.languagePack.getMonsterStrings(ID);
+    public static final String NAME = monsterStrings.NAME;
+    public static final String[] MOVES = monsterStrings.MOVES;
+    private static final String IMG = SpireAnniversary6Mod.makeImagePath("monsters/Frostlands/Spruce.png");
+    public int move = 0;
+    private static final byte FIRST_MOVE = 0, PRIMARY = 1, SECONDARY = 2;
+    private static final int HP = 30, HP_MAX = 40, DMG = 12, A7_DMG = 14, BLK = 8, DMG2 = 8;
+
+    public Spruce() {
+        this(0.0f, 0.0f);
+    }
+
+    public Spruce(final float x, final float y) {
+        super(Spruce.NAME, ID, HP, 20.0F, -10.0F, 190.0F, 235, IMG, x, y);
+        loadAnimation(modID + "Resources/images/monsters/Frostlands/Spruce.atlas", modID + "Resources/images/monsters/Frostlands/Spruce.json", 1.75F);
+        state.setAnimation(0, "idle", true);
+        type = EnemyType.NORMAL;
+        if (AbstractDungeon.ascensionLevel >= 7)
+        {
+            setHp(HP+8, HP_MAX+8);
+            damage.add(new DamageInfo(this, A7_DMG));
+        }
+        else
+        {
+            setHp(HP, HP_MAX);
+            damage.add(new DamageInfo(this, DMG));
```

Comment:
HP and damage modification doN#t come from the smae asc

Reply:
Fixed